py-polars/polars/frame.py

```diff
@@ -41,6 +43,15 @@
     from .lazy import LazyFrame, Expr
 
 
+def to_csv(df, path_or_buf=None):
```

Comment:
Could you put the logic in `DataFrame.to_csv` and remove this function? I want the API similar to the rest of IO.

Reply:
Modified, please review. Thank you for your guidance.

Reply:
Modified, please review. Thank you for your guidance.